conference.js

```diff
@@ -2112,6 +2113,10 @@ export default {
             const localAudio = getLocalJitsiAudioTrack(APP.store.getState());
             let newLvl = lvl;
 
+            if (this.isLocalId(id)) {
```

Comment:
I think this may still trigger if we are muted (if the track exists of course). Can you double check?

Reply:
So I checked and JitsiConferenceEvents.TRACK_AUDIO_LEVEL_CHANGED is not triggered while muted.